tests/director/tests/formation_mapping_api_test.go

```diff
@@ -345,6 +351,20 @@ func Test_UpdateStatus(baseT *testing.T) {
 		// Create Application Template
 		appTemplateName := createAppTemplateName("app-template-name-subscription-async")
 		appTemplateInput := fixtures.FixApplicationTemplateWithoutWebhooks(appTemplateName)
+		placeholderName := "name"
+		placeholderDisplayName := "display-name"
+		appTemplateInput.Placeholders = []*graphql.PlaceholderDefinitionInput{
+			{
+				Name:        "name",
+				Description: &placeholderName,
+				JSONPath:    &conf.TenantProviderConfig.SubscriptionProviderAppNameProperty,
+			},
+			{
+				Name:        "display-name",
+				Description: &placeholderDisplayName,
+				JSONPath:    &conf.TenantProviderConfig.SubscriptionProviderAppNameProperty,
+			},
```

Comment:
Why this is needed? We already have the `Name`, `Description` and `JSONPath` properties set in the fixture above - `fixtures.FixApplicationTemplateWithoutWebhooks`?

Reply:
I thought will be clear to replace the whole array, will replace only the JSONPath property here, as it is different than the one created with fix-function.